### ISSUE TITLE
Degree is letter-like and not an operator

### ISSUE THREAD

**Issue by rocky** [rsmenon/pygments-mathematica#13]
\[Degree] is letter-like, not an operator,

Therefore, when we write 360° that is analogous to 2π (pun intended)  it this interpreted as: 360 times "degree", and analogous to: 2 times "pi".

Note also that writing: \[Degree] == Degree  is valid, and this would not be the case if this symbol were not letterlike.

**Comment by rsmenon** [comment]
You're right, it should be a symbol, not an operator. I will batch this change along with a few others.
